app/game/typingGame/script.js

```diff
@@ -0,0 +1,56 @@
+//Maybe replace spaceQuotes with random quote API?
```

Comment:
Yeah you're right. We're probably gonna store some quotes or something in a json file in the backend and just send it to the frontend